meta-ostro/conf/distro/include/ostroproject-ci.inc

```diff
@@ -31,27 +31,86 @@ SDKMACHINE ?= "x86_64"
 # Generate source mirror tarballs:
 BB_GENERATE_MIRROR_TARBALLS = "1"
 
-# Include the devel settings
+# If (and only if) building in the CI, the BUILD_ID is guaranteed to
+# be consistent across different builds and therefore it can and
+# should be included in the os-release package.
+#
+# When doing the same outside of the CI, we'd end up with an
+# os-release file that has BUILD_ID = ${DATETIME} for the first time
+# that the os-release file was generated. It does not get re-generated
+# (explicit vardepsexclude in os-release.bb) because that would cause
+# constant rebuilds.
+OS_RELEASE_FIELDS_append = " BUILD_ID"
+
+# Configure images in "development mode", but without the additional
+# packages in the ostro-image itself. Instead, swupd images are enabled
+# below.
 require conf/distro/include/ostro-os-development.inc
+OSTRO_DEVELOPMENT_EXTRA_FEATURES = ""
+OSTRO_DEVELOPMENT_EXTRA_INSTALL = ""
 
 # XXX Drop the VM hack after taking care also of the non UEFI devices
 # (those using U-Boot: edison and beaglebone)
 OSTRO_VM_IMAGE_TYPES = "dsk.xz dsk.zip dsk.vdi.zip"
 
+# Temporarily disable Edison "toflash" image format
+# because depends on a .rootfs.manifest that is not gettting
+# created for swupd images. Looks like we need to create that
+# file after all, the security checks also want it.
+IMAGE_FSTYPES_remove_pn-edison = "toflash"
```

Comment:
Ignore this please. It can already be removed again and I will do so in the next force push.